lib/vsc/utils/timestamp.py

```diff
@@ -44,7 +44,7 @@
 
 def convert_to_datetime(timestamp=None):
     """
-    Convert a string or datetime.datime instance to a datetime.datetime with local tzinfo
+    Convert a string or datetime.datime instance to a datetime.datetime with UTC tzinfo
```

Comment:
this will not break anything else?

Reply:
it's a bug. if fixing the bug breaks other stuff, should fix the other bugs too